web/src/app/components/home/home.component.ts

```diff
@@ -14,22 +16,23 @@ export class HomeComponent implements OnInit {
   ngOnInit() {
     this.isResponseLoading = true;
     this.isLogin = false;
+    this.userService.getUsersWithMaxFollowers()
+      .subscribe((tribeUsers: UserSocial[]) => {
+        if (tribeUsers.length > 0) {
```

Comment:
Probably cleaner with a `filter` inside a `pipe`

Reply:
cool. Will change it 👍